screens/Home/MyVcsTabMachine.ts

```diff
@@ -86,7 +86,10 @@ export const MyVcsTabMachine = model.createMachine(
         states: {
           checkStorage: {
             invoke: {
-              src: () => Promise.resolve(isMaximumStorageLimitReached()),
+              src: () =>
+                Promise.resolve(
+                  Storage.isMinimumLimitReached('minimumStorageRequiredInMB')
```

Comment:
do we need to mention inMB here? we have comments added for every property in the mosip-config. we can do the same for this property as well.

Reply:
Refactored